.github/workflows/somehow-work.yaml

```
@@ -0,0 +1,68 @@
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  x86-build-and-push:
+    runs-on: ubuntu-latest  # Using GitHub-hosted runner instead of self-hosted
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch }}
```

actionlint (1.7.4)

property "branch" is not defined in object type {}

(expression)